pkg/operation/botanist/operatingsystemconfig.go

```diff
@@ -312,6 +312,9 @@ func (b *Botanist) deployOperatingSystemConfigsForWorker(machineTypes []gardenco
 		"kubeletDataVolume": worker.KubeletDataVolumeName,
 	}
 
+	originalConfig["enableContainerD"] = worker.Cri != nil && worker.Cri.EnableContainerD
+	downloaderConfig["enableContainerD"] = worker.Cri != nil && worker.Cri.EnableContainerD
```

Comment:
I think the downloader does not care about the container runtime, but I have to double check it.